gradle.properties

```diff
@@ -1,5 +1,5 @@
 GROUP=com.apollographql.android
-VERSION_NAME=0.2.1-SNAPSHOT
+VERSION_NAME=0.2.2-SNAPSHOT
```

Comment:
I dont think this is necessary if you do  ./gradlew build **--refresh dependencies**